src/lib.rs

```diff
@@ -109,6 +105,15 @@ use vec1::Vec1;
 pub type Result<T> = std::result::Result<T, IronOxideErr>;
 type PolicyCache = DashMap<PolicyGrant, Vec<WithKey<UserOrGroup>>>;
 
+// This is where we export structs that don't fit into a single module.
+// They were previously exported at the top level, but added clutter to the docs landing page.
+/// Types that are useful in multiple modules
+pub mod common {
+    pub use crate::internal::{
+        DeviceContext, DeviceSigningKeyPair, PrivateKey, PublicKey, SdkOperation,
```

Comment:
I would like to get more top-level types into here (like `PrivateKeyRotationCheckResult`), but that's not really a "common" struct, so we'd need to re-word/purpose the module.